pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+## Links
+
+- [Jira task]()
+- [Upstream repository]()
+- [Changelog/diff]()
+- [Related PR(s)]()
+
+--------------------
+
+## Description
+
+- Target channel: 
+- Package version: 
+### Dependency chain
+N/A
+
+### Information checked from upstream: 
+
+- [ ] All dependency pinnings match with upstream.
+- [ ] All URLs are correct.
+- [ ] abs.yaml is not present at time of review.
+### Recipe changes:
```

Comment:
`Recipe changes and rationales`? It would be great to encourage people to write down _why_ they've made the changes, because as a reviewer I often spend a lot of time trying to figure this out.